alpaca_trade_api/stream.py

```diff
@@ -67,7 +67,9 @@ def __init__(self,
     async def _connect(self):
         self._ws = await websockets.connect(
             self._endpoint,
-            extra_headers={'Content-Type': 'application/msgpack'})
+            extra_headers={'Content-Type': 'application/msgpack'},
+            ping_timeout=180,
+        )
```

Comment:
This should probably also set `ping_interval` to 10? Looks like the default is 20 seconds

Reply:
That doesn't really matter, it doesn't have to be symmetrical.